src/Library/demos/File Monitor/main.js

```diff
@@ -0,0 +1,75 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+import Adw from "gi://Adw";
+import GLib from "gi://GLib";
+
+Gio._promisify(Gtk.FileLauncher.prototype, "launch", "launch_finish");
+
```

Comment:
I guess we should add the other `Gio.File` operations? They're probably already done though, so I'm not sure.

Reply:
I think most of them have been covered, wouldn't be worth adding them. We can add it back if that's needed